tail.go

```diff
@@ -315,7 +313,8 @@ func (tail *Tail) tailFileSync() {
 				return
 			}
 		} else {
-			// non-EOF error
+			// Send a signal to caller asking a shutdown
+			tail.Lines <- &Line{Err: err}
 			tail.Killf("Error reading %s: %s", tail.Filename, err)
```

Comment:
why ask for a shutdown?

Reply:
idea is to kill the goroutine and retry in next cycle. in general case tail just returns and completes.

Reply:
it closes the Lines channel before returning, check that instead